docs/csharp/linq/get-started/walkthrough-writing-queries-linq.md

```diff
@@ -29,7 +29,7 @@ The first step is to create a data source for your queries. The data source for
 
 Note the following characteristics:
 
-- The `Student` record consists of autoimplemented properties.
+- The `Student` record consists of automatically implemented properties.
```

Comment:
Does this mean that the phrase `auto-implemented` is no longer a thing?

Reply:
Yes. The terminology we want to use consistently is "automatically implemented"